### ISSUE TITLE
[$1000 bounty] Kia EV9 lateral port

### ISSUE THREAD

**Issue by sshane** [commaai/opendbc#1175]
This bounty is for a lateral only port merged into master with a longitudinal proof of concept (ECU disable or intercept if the camera sends longitudinal commands)

- We should understand the lateral API such that we can guarantee some maximum lateral acceleration at all speeds
- No faults/unexpected alerts from the car
- We should allow smooth and safe user steering blending

**Comment by sunnyhaibin** [comment]
https://github.com/commaai/opendbc/pull/1674

**Comment by zhaog100** [comment]
I'm claiming this bounty. I'll work on the Kia EV9 lateral port.

**Comment by MyTH-zyxeon** [comment]
Quick status/coordination ping on the $1000 EV9 lateral port bounty (#1175):

- There is an existing draft PR that looks like it targets EV9 steering support: #1674 ("HKG: Car Port for Kia EV9 2024 (HDA2 & LFA2)").
- Also saw a public "claiming this bounty" comment from @zhaog100.

Questions:
1) Is #1674 the intended solution path for this bounty, or is the bounty still open for a fresh PR?
2) If #1674 is the right path, are there specific review/CI/DBC cleanup tasks that a contributor *without* vehicle/hardware access can help with to unblock it?

If you point to the preferred path, I can help by doing static review on the DBC changes, proposing safety/steering-limit checks, and writing/adding unit tests where possible (no live vehicle testing).

**Comment by jespinosa1983-cyber** [comment]
/claim

Payout public ID:
- Network: USDC/Polygon
- Address/ID: 0x014F9cda483A46D518a154291007860e56CEaB3B

**Comment by jespinosa1983-cyber** [comment]
/claim

Payout public ID:
- Network: USDC/Polygon
- Address/ID: 0x014F9cda483A46D518a154291007860e56CEaB3B
